### ISSUE TITLE
No audio channel when merging video with audio

### ISSUE THREAD

**Issue by truroshan** [savonet/liquidsoap#3840]
**Describe the bug**
When attempting to merge a video with audio using the provided script, the output video does not contain the audio channel.
**To Reproduce**
A minimal script to reproduce the issue is the most efficient way to have your bug addressed and fixed very quickly!

**Expected behavior**
merged video stream (s) should contain both the audio and video channels. 

**Code**
```
log.level := 4 

audio = once(single("./audio.mp3"))

video = single("./video.mp4")

stream = source(source.tracks(audio).{video=source.tracks(video).video})

clock.assign_new(sync='none',[stream])

enc = %ffmpeg(
        format="flv",
        %audio.copy,
        %video.copy
        )

output.file(fallible=true,on_stop=shutdown,
             enc, "copy.mp4",
             stream)


```

**Version details**

- OS: NixOS
- Version 23.11
- Liquidsoap 2.2.4-1+git@nixpkgs

**Install method**
nix package manager 
https://search.nixos.org/packages?channel=unstable&show=liquidsoap&from=0&size=50&sort=relevance&type=packages&query=liquidsoap

**Common issues**
No audio in output video when merging video and audio.

logs
```
INFO: Loading Sdl_image, Target = linux
INFO: Loading Sdl_ttf, Target = linux
2024/03/31 21:30:36 >>> LOG START
2024/03/31 21:30:30 [ffmpeg.filter.bitstream:3] No valid mode found for filter pgs_frame_merge!
2024/03/31 21:30:30 [main:3] Liquidsoap 2.2.4-1+git@nixpkgs
2024/03/31 21:30:30 [main:3] Using: alsa=0.3.0 angstrom=0.16.0 ao=0.2.4 bigarray=[distributed with OCaml] bigarray-compat=1.1.0 bigstringaf=0.9.0 bjack=0.1.6 bytes=[distributed with OCaml] camlimages.all_formats=4.2.6 camlimages.core=5.0.4 camlimages.exif=5.0.4 camlimages.gif=5.0.4 camlimages.jpeg=5.0.4 camlimages.png=5.0.4 camlimages.tiff=5.0.4 camlimages.xpm=5.0.4 camlp-streams camomile.lib=2.0 checkseum.ocaml=0.5.2 cry=1.0.3 ctypes=0.22.0 ctypes-foreign=0.22.0 ctypes.stubs=0.22.0 curl=0.9.2 decompress.de=1.5.3 decompress.zl=1.5.3 dssi=0.1.5 dtools=0.4.5 dune-build-info=3.14.2 dune-private-libs.dune-section=3.14.2 dune-site=3.14.2 dune-site.private=3.14.2 duppy=0.9.3 equeue faad=0.5.1 fdkaac=0.3.3 ffmpeg-av=1.1.8 ffmpeg-avcodec=1.1.8 ffmpeg-avdevice=1.1.8 ffmpeg-avfilter=1.1.8 ffmpeg-avutil=1.1.8 ffmpeg-swresample=1.1.8 ffmpeg-swscale=1.1.8 fileutils=0.6.4 flac=0.5.0 flac.decoder=0.5.0 flac.ogg=0.5.0 frei0r=0.1.2 gd=1.0a5 gen=1.1 graphics=5.1.2 imagelib=20221222 imagelib.unix=20221222 inotify=2.5 integers ladspa=0.2.2 lame=0.3.7 lastfm=0.3.3 lilv=0.1.0 liquidsoap-lang=2.2.4-1 liquidsoap-lang.console=2.2.4-1 liquidsoap_alsa liquidsoap_ao liquidsoap_bjack liquidsoap_builtins liquidsoap_camlimages liquidsoap_core liquidsoap_dssi liquidsoap_faad liquidsoap_fdkaac liquidsoap_ffmpeg liquidsoap_flac liquidsoap_frei0r liquidsoap_gd liquidsoap_graphics liquidsoap_imagelib liquidsoap_ladspa liquidsoap_lame liquidsoap_lastfm liquidsoap_lilv liquidsoap_lo liquidsoap_mad liquidsoap_ogg liquidsoap_ogg_flac liquidsoap_optionals liquidsoap_opus liquidsoap_oss liquidsoap_portaudio liquidsoap_posix_time liquidsoap_pulseaudio liquidsoap_runtime liquidsoap_samplerate liquidsoap_sdl liquidsoap_shine liquidsoap_soundtouch liquidsoap_speex liquidsoap_srt liquidsoap_ssl liquidsoap_stereotool liquidsoap_taglib liquidsoap_theora liquidsoap_vorbis liquidsoap_xmlplaylist liquidsoap_yaml lo=0.2.0 mad=0.5.2 magic-mime=1.3.1 menhirLib=20230608 metadata=0.2.0 mm=0.8.5 mm.audio=0.8.5 mm.base=0.8.5 mm.image=0.8.5 mm.midi=0.8.5 mm.video=0.8.5 netclient netstring netsys ogg=0.7.4 ogg.decoder=0.7.4 optint=0.3.0 opus=0.2.2 opus.decoder=0.2.2 pcre=7.5.0 portaudio=0.2.3 posix-base=2.0.2 posix-socket=2.0.2 posix-socket.constants=2.0.2 posix-socket.stubs=2.0.2 posix-socket.types=2.0.2 posix-time2=2.0.2 posix-time2.constants=2.0.2 posix-time2.stubs=2.0.2 posix-time2.types=2.0.2 posix-types=2.0.2 posix-types.constants=2.0.2 pulseaudio=0.1.5 result=1.5 samplerate=0.1.6 sedlex=3.2 seq=[distributed with OCaml 4.07 or above] shine=0.2.3 soundtouch=0.1.9 speex=0.4.1 speex.decoder=0.4.1 srt=0.3.0 srt.constants=0.3.0 srt.stubs=0.3.0 srt.stubs.locked=0.3.0 srt.types=0.3.0 ssl=0.7.0 stdlib-shims=0.3.0 stereotool str=[distributed with OCaml] stringext=1.6.0 taglib=0.3.10 theora=0.4.0 theora.decoder=0.4.0 threads=[distributed with OCaml] threads.posix=[distributed with OCaml] tsdl=v1.0.0 tsdl-image=0.5 tsdl-ttf=0.5 unix=[distributed with OCaml] unix-errno=0.6.2 unix-errno.errno_bindings=0.6.2 unix-errno.errno_types=0.6.2 unix-errno.errno_types_detected=0.6.2 unix-errno.unix=0.6.2 uri=4.2.0 vorbis=0.8.0 vorbis.decoder=0.8.0 xmlm=1.4.0 xmlplaylist=0.1.5 yaml=3.2.0 yaml.bindings=3.2.0 yaml.bindings.types=3.2.0 yaml.c=3.2.0 yaml.ffi=3.2.0 yaml.types=3.2.0
2024/03/31 21:30:30 [main:3]
2024/03/31 21:30:30 [main:3] DISCLAIMER: This version of Liquidsoap has been compiled from a snapshot of the
2024/03/31 21:30:30 [main:3] development code. As such, it should not be used in production unless you know
2024/03/31 21:30:30 [main:3] what you are doing!
2024/03/31 21:30:30 [main:3]
2024/03/31 21:30:30 [main:3] We are, however, very interested in any feedback about our development code and
2024/03/31 21:30:30 [main:3] committed to fix issues as soon as possible.
2024/03/31 21:30:30 [main:3]
2024/03/31 21:30:30 [main:3] If you are interested in collaborating to the development of Liquidsoap, feel
2024/03/31 21:30:30 [main:3] free to drop us a mail at <savonet-devl@lists.sf.net> or to join the slack chat
2024/03/31 21:30:30 [main:3] at <http://slack.liquidsoap.info>.
2024/03/31 21:30:30 [main:3]
2024/03/31 21:30:30 [main:3] Please send any bug report or feature request at
2024/03/31 21:30:30 [main:3] <https://github.com/savonet/liquidsoap/issues>.
2024/03/31 21:30:30 [main:3]
2024/03/31 21:30:30 [main:3] We hope you enjoy this snapshot build of Liquidsoap!
2024/03/31 21:30:30 [main:3]
2024/03/31 21:30:30 [clock:3] Using native (high-precision) implementation for latency control
2024/03/31 21:30:36 [main:3] Standard library loaded in 5.19 seconds.
2024/03/31 21:30:36 [request.0:4] Pushed ["./audio.mp3";...].
2024/03/31 21:30:36 [request.1:4] Pushed ["./video.mp4";...].
2024/03/31 21:30:36 [clock:4] Currently 1 clock(s) allocated.
2024/03/31 21:30:36 [frame:4] frame.audio.channels set to: 2
2024/03/31 21:30:36 [frame:4] frame.video.default set to: false
2024/03/31 21:30:36 [frame:4] frame.midi.channels set to: 0
2024/03/31 21:30:36 [frame:4] frame.video.width set to: 1280
2024/03/31 21:30:36 [frame:4] frame.video.height set to: 720
2024/03/31 21:30:36 [frame:4] frame.audio.samplerate set to: 44100
2024/03/31 21:30:36 [frame:4] frame.video.framerate set to: 25
2024/03/31 21:30:36 [frame:4] frame.audio.samplerate set to: 44100
2024/03/31 21:30:36 [frame:3] Using 44100Hz audio, 25Hz video, 44100Hz main.
2024/03/31 21:30:36 [frame:3] Video frame size set to: 1280x720
2024/03/31 21:30:36 [frame:3] Frame size must be a multiple of 1764 ticks = 1764 audio samples = 1 video samples.
2024/03/31 21:30:36 [frame:3] Targeting 'frame.duration': 0.04s = 1764 audio samples = 1764 ticks.
2024/03/31 21:30:36 [frame:3] Frames last 0.04s = 1764 audio samples = 1 video samples = 1764 ticks.
2024/03/31 21:30:36 [sandbox:3] Sandboxing disabled
2024/03/31 21:30:36 [startup:3] DSSI plugins registration: 0.00s
2024/03/31 21:30:36 [startup:3] FFmpeg filters registration: 0.03s
2024/03/31 21:30:36 [startup:3] FFmpeg bitstream filters registration: 0.00s
2024/03/31 21:30:36 [startup:3] Lilv plugins registration: 0.00s
2024/03/31 21:30:36 [startup:3] Frei0r plugin registration: 0.00s
2024/03/31 21:30:36 [startup:3] LADSPA plugins registration: 0.00s
2024/03/31 21:30:36 [startup:3] Typechecking: 4.86s
2024/03/31 21:30:36 [startup:3] Evaluation: 0.02s
2024/03/31 21:30:36 [startup:3] Typechecking: 0.04s
2024/03/31 21:30:36 [startup:3] Evaluation: 0.00s
2024/03/31 21:30:36 [startup:3] Typechecking: 0.01s
2024/03/31 21:30:36 [startup:3] Evaluation: 0.00s
2024/03/31 21:30:36 [startup:3] Loaded final.liq: 0.01s
2024/03/31 21:30:36 [clock.source:4] Starting source(s): output.file
2024/03/31 21:30:36 [source:4] Source output.file gets up with content type: {audio=ffmpeg.copy,video=ffmpeg.copy}.
2024/03/31 21:30:36 [output_file:3] Content type is {audio=ffmpeg.copy,video=ffmpeg.copy}.
2024/03/31 21:30:36 [source:4] Source source gets up with content type: {audio=ffmpeg.copy,video=ffmpeg.copy}.
2024/03/31 21:30:36 [source:3] Content type is {audio=ffmpeg.copy,video=ffmpeg.copy}.
2024/03/31 21:30:36 [source:4] Source single.2 gets up with content type: {video=ffmpeg.copy}.
2024/03/31 21:30:36 [single.2:3] "./video.mp4" is static, resolving once for all...
2024/03/31 21:30:36 [decoder.ogg:4] Unsupported MIME type for "./video.mp4": video/mp4!
2024/03/31 21:30:36 [decoder.ffmpeg:4] Unsupported file extension for "./video.mp4"!
2024/03/31 21:30:36 [decoder.mad:4] Unsupported MIME type for "./video.mp4": video/mp4!
2024/03/31 21:30:36 [decoder.flac:4] Unsupported MIME type for "./video.mp4": video/mp4!
2024/03/31 21:30:36 [decoder.mp4:4] Unsupported file extension for "./video.mp4"!
2024/03/31 21:30:36 [decoder.mp4:4] Unsupported MIME type for "./video.mp4": video/mp4!
2024/03/31 21:30:36 [decoder.aac:4] Unsupported MIME type for "./video.mp4": video/mp4!
2024/03/31 21:30:36 [decoder.midi:4] Unsupported MIME type for "./video.mp4": video/mp4!
2024/03/31 21:30:36 [decoder.srt:4] Unsupported MIME type for "./video.mp4": video/mp4!
2024/03/31 21:30:36 [decoder.aiff:4] Unsupported MIME type for "./video.mp4": video/mp4!
2024/03/31 21:30:36 [decoder.wav:4] Unsupported MIME type for "./video.mp4": video/mp4!
2024/03/31 21:30:36 [decoder:4] Available decoders: ffmpeg (priority: 10), mp4 (priority: 1), image (priority: 1)
2024/03/31 21:30:36 [decoder:4] Trying decoder "ffmpeg"
2024/03/31 21:30:36 [decoder.ffmpeg:3] Requested content-type for "./video.mp4": {video=ffmpeg.copy}
2024/03/31 21:30:36 [decoder.ffmpeg:3] FFmpeg recognizes "./video.mp4" as audio: {codec: aac, 44100Hz, 2 channel(s)}, video: {codec: h264, 1920x1080, yuv420p}
2024/03/31 21:30:36 [decoder.ffmpeg:3] Decoded content-type for "./video.mp4": {video=ffmpeg.copy(codec="h264",width=1920,height=1080,aspect_ratio=1/1,pixel_format=yuv420p)}
2024/03/31 21:30:36 [decoder:4] Selected decoder ffmpeg for file "./video.mp4" with expected kind {video=ffmpeg.copy(codec="h264",width=1920,height=1080,aspect_ratio=1/1,pixel_format=yuv420p)} and detected content {video=ffmpeg.copy(codec="h264",width=1920,height=1080,aspect_ratio=1/1,pixel_format=yuv420p)}
2024/03/31 21:30:36 [decoder.video.metadata:4] Unsupported file extension for "./video.mp4"!
2024/03/31 21:30:36 [decoder.ogg.metadata:4] Unsupported MIME type for "./video.mp4": video/mp4!
2024/03/31 21:30:36 [decoder.image.metadata:4] Unsupported MIME type for "./video.mp4": video/mp4!
2024/03/31 21:30:36 [decoder.id3:4] Unsupported MIME type for "./video.mp4": video/mp4!
2024/03/31 21:30:36 [decoder.id3:4] Unsupported MIME type for "./video.mp4": video/mp4!
2024/03/31 21:30:36 [decoder.id3:4] Unsupported MIME type for "./video.mp4": video/mp4!
2024/03/31 21:30:36 [decoder.flac.metadata:4] Unsupported MIME type for "./video.mp4": video/mp4!
2024/03/31 21:30:36 [metadata.mp4:4] Unsupported file extension for "./video.mp4"!
2024/03/31 21:30:36 [metadata.mp4:4] Unsupported MIME type for "./video.mp4": video/mp4!
2024/03/31 21:30:36 [decoder.ffmpeg:4] Unsupported file extension for "./video.mp4"!
2024/03/31 21:30:36 [metadata.flac:4] Unsupported MIME type for "./video.mp4": video/mp4!
2024/03/31 21:30:36 [decoder.ogg:4] Unsupported MIME type for "./video.mp4": video/mp4!
2024/03/31 21:30:36 [decoder.taglib:4] Unsupported file extension for "./video.mp4"!
2024/03/31 21:30:36 [source:4] Source once gets up with content type: {audio=ffmpeg.copy}.
2024/03/31 21:30:36 [source:4] Source single gets up with content type: {audio=ffmpeg.copy}.
2024/03/31 21:30:36 [single:3] "./audio.mp3" is static, resolving once for all...
2024/03/31 21:30:36 [decoder.ogg:4] Unsupported MIME type for "./audio.mp3": audio/mpeg!
2024/03/31 21:30:36 [decoder.ffmpeg:4] Unsupported file extension for "./audio.mp3"!
2024/03/31 21:30:36 [decoder.mad:4] Unsupported file extension for "./audio.mp3"!
2024/03/31 21:30:36 [decoder.flac:4] Unsupported MIME type for "./audio.mp3": audio/mpeg!
2024/03/31 21:30:36 [decoder.mp4:4] Unsupported MIME type for "./audio.mp3": audio/mpeg!
2024/03/31 21:30:36 [decoder.aac:4] Unsupported MIME type for "./audio.mp3": audio/mpeg!
2024/03/31 21:30:36 [decoder.midi:4] Unsupported MIME type for "./audio.mp3": audio/mpeg!
2024/03/31 21:30:36 [decoder.srt:4] Unsupported MIME type for "./audio.mp3": audio/mpeg!
2024/03/31 21:30:36 [decoder.aiff:4] Unsupported MIME type for "./audio.mp3": audio/mpeg!
2024/03/31 21:30:36 [decoder.wav:4] Unsupported MIME type for "./audio.mp3": audio/mpeg!
2024/03/31 21:30:36 [decoder:4] Available decoders: ffmpeg (priority: 10), mad (priority: 1), image (priority: 1)
2024/03/31 21:30:36 [decoder:4] Trying decoder "ffmpeg"
2024/03/31 21:30:36 [decoder.ffmpeg:3] Requested content-type for "./audio.mp3": {audio=ffmpeg.copy}
2024/03/31 21:30:36 [decoder.ffmpeg:3] FFmpeg recognizes "./audio.mp3" as audio: {codec: mp3, 48000Hz, 2 channel(s)}
2024/03/31 21:30:36 [decoder.ffmpeg:3] Decoded content-type for "./audio.mp3": {audio=ffmpeg.copy(codec="mp3",channel_layout="stereo",sample_format=fltp,sample_rate=48000)}
2024/03/31 21:30:36 [decoder:4] Selected decoder ffmpeg for file "./audio.mp3" with expected kind {audio=ffmpeg.copy(codec="mp3",channel_layout="stereo",sample_format=fltp,sample_rate=48000)} and detected content {audio=ffmpeg.copy(codec="mp3",channel_layout="stereo",sample_format=fltp,sample_rate=48000)}
2024/03/31 21:30:36 [decoder.video.metadata:4] Unsupported MIME type for "./audio.mp3": audio/mpeg!
2024/03/31 21:30:36 [decoder.ogg.metadata:4] Unsupported MIME type for "./audio.mp3": audio/mpeg!
2024/03/31 21:30:36 [decoder.image.metadata:4] Unsupported MIME type for "./audio.mp3": audio/mpeg!
2024/03/31 21:30:36 [decoder.id3:4] Unsupported file extension for "./audio.mp3"!
2024/03/31 21:30:36 [decoder.id3:4] Unsupported file extension for "./audio.mp3"!
2024/03/31 21:30:36 [decoder.id3:4] Unsupported file extension for "./audio.mp3"!
2024/03/31 21:30:36 [decoder.flac.metadata:4] Unsupported MIME type for "./audio.mp3": audio/mpeg!
2024/03/31 21:30:36 [metadata.mp4:4] Unsupported MIME type for "./audio.mp3": audio/mpeg!
2024/03/31 21:30:36 [decoder.ffmpeg:4] Unsupported file extension for "./audio.mp3"!
2024/03/31 21:30:36 [metadata.flac:4] Unsupported MIME type for "./audio.mp3": audio/mpeg!
2024/03/31 21:30:36 [decoder.ogg:4] Unsupported MIME type for "./audio.mp3": audio/mpeg!
2024/03/31 21:30:36 [decoder.taglib:4] Unsupported file extension for "./audio.mp3"!
2024/03/31 21:30:36 [source:4] Source source.fail gets up with content type: {audio=ffmpeg.copy(codec="mp3",channel_layout="stereo",sample_format=fltp,sample_rate=48000)}.
2024/03/31 21:30:36 [source.fail:3] Content type is {audio=ffmpeg.copy(codec="mp3",channel_layout="stereo",sample_format=fltp,sample_rate=48000)}.
2024/03/31 21:30:36 [threads:4] Created thread "clock_source" (1 total).
2024/03/31 21:30:36 [clock:4] Main phase starts.
2024/03/31 21:30:36 [threads:4] Created thread "generic queue #1" (1 total).
2024/03/31 21:30:36 [threads:4] Created thread "generic queue #2" (2 total).
2024/03/31 21:30:36 [threads:4] Created thread "non-blocking queue #1" (3 total).
2024/03/31 21:30:36 [threads:4] Created thread "non-blocking queue #2" (4 total).
2024/03/31 21:30:36 [video.converter:3] Using preferred video converter: ffmpeg.
2024/03/31 21:30:36 [audio.converter:3] Using samplerate converter: libsamplerate.
2024/03/31 21:30:36 [video.text:3] Using sdl implementation
2024/03/31 21:30:36 [clock.source:3] Streaming loop starts in no sync mode
2024/03/31 21:30:36 [clock.source:3] Delegating synchronization to active sources
2024/03/31 21:30:36 [_/video_mp4:3] Prepared "./video.mp4" (RID 1).
2024/03/31 21:30:36 [_/audio_mp3:3] Prepared "./audio.mp3" (RID 0).
2024/03/31 21:30:36 [request.0:4] Currently on air.
2024/03/31 21:30:36 [request.1:4] Currently on air.
2024/03/31 21:30:36 [decoder:2] Decoding "./video.mp4" ended: Ffmpeg_decoder.End_of_file.
2024/03/31 21:30:36 [decoder:4] Raised at Ffmpeg_decoder.mk_decoder.(fun).f in file "src/core/decoder/ffmpeg_decoder.ml", line 890, characters 12-29
2024/03/31 21:30:36 [decoder:4] Called from Decoder.mk_decoder.fill in file "src/core/decoder/decoder.ml", line 505, characters 10-31
2024/03/31 21:30:36 [decoder:4]
2024/03/31 21:30:36 [_/video_mp4:4] Finished with "./video.mp4".
2024/03/31 21:30:36 [_/video_mp4:3] Prepared "./video.mp4" (RID 1).
2024/03/31 21:30:36 [request.1:4] Currently on air.
2024/03/31 21:30:37 [decoder:2] Decoding "./audio.mp3" ended: Ffmpeg_decoder.End_of_file.
2024/03/31 21:30:37 [decoder:4] Raised at Ffmpeg_decoder.mk_decoder.(fun).f in file "src/core/decoder/ffmpeg_decoder.ml", line 890, characters 12-29
2024/03/31 21:30:37 [decoder:4] Called from Decoder.mk_decoder.fill in file "src/core/decoder/decoder.ml", line 505, characters 10-31
2024/03/31 21:30:37 [decoder:4]
2024/03/31 21:30:37 [_/audio_mp3:4] Finished with "./audio.mp3".
2024/03/31 21:30:37 [output_file:3] Source failed (no more tracks) stopping output...
2024/03/31 21:30:37 [threads:3] Main loop exited
2024/03/31 21:30:37 [main:3] Shutdown started!
2024/03/31 21:30:37 [threads:3] Waiting for main threads to terminate...
2024/03/31 21:30:37 [threads:4] Waiting for thread clock_source to shutdown
2024/03/31 21:30:37 [source:4] Source output_file gets down.
2024/03/31 21:30:37 [source:4] Source source gets down.
2024/03/31 21:30:37 [source:4] Source _/video_mp4 gets down.
2024/03/31 21:30:37 [_/video_mp4:4] Finished with "./video.mp4".
2024/03/31 21:30:37 [source:4] Source once gets down.
2024/03/31 21:30:37 [source:4] Source _/audio_mp3 gets down.
2024/03/31 21:30:37 [source:4] Source source.fail gets down.
2024/03/31 21:30:37 [clock.source:3] Streaming loop stopped.
2024/03/31 21:30:37 [threads:4] Thread "clock_source" terminated (0 remaining).
2024/03/31 21:30:37 [threads:3] Main threads terminated.
2024/03/31 21:30:37 [threads:3] Shutting down scheduler...
2024/03/31 21:30:37 [threads:4] Thread "generic queue #2" terminated (3 remaining).
2024/03/31 21:30:37 [threads:4] Thread "non-blocking queue #1" terminated (2 remaining).
2024/03/31 21:30:37 [threads:4] Thread "generic queue #1" terminated (1 remaining).
2024/03/31 21:30:37 [threads:4] Thread "non-blocking queue #2" terminated (0 remaining).
2024/03/31 21:30:37 [threads:3] Scheduler shut down.
2024/03/31 21:30:37 [main:3] Cleaning downloaded files...
2024/03/31 21:30:37 [request.0:4] Request finished.
2024/03/31 21:30:37 [request.1:4] Request finished.
2024/03/31 21:30:37 [main:3] Freeing memory...
2024/03/31 21:30:37 >>> LOG END
```

**Comment by gAlleb** [comment]
Something's wrong in the script. 

Anyway, please use something like this:

```
audio = once(single("./audio.mp3"))
video = single("./video.mp4")

stream = source(source.tracks(audio).{video=source.tracks(video).video})
clock.assign_new(sync='none',[stream])

enc =
  %ffmpeg(
    format = "flv",
    %video(
      codec = "libx264", b = "3000k", maxrate = "5000k", bufsize = "10000k", g = "50",preset = "ultrafast", tune="zerolatency", pixel_format = "yuv420p"),
    %audio(codec="aac",
        samplerate=48000,
        channels=2,
        b="128k",
        profile="aac_low")
  )

output.file(fallible=true,on_stop=shutdown,
             enc, "copy.mp4",
             stream)

```

**Comment by toots** [comment]
I think that they want to audio/video copy, not encoding.

I'll see if I can reproduce soon. One quick thing I noticed: the format is set to flash (`flv`) but the file extension is set to `.mp4`. One of the two will be picked but I'm not sure which.

**Comment by truroshan** [comment]
no problem use .mp4

yes sir, i don't want re-encoding.

**Comment by toots** [comment]
I can confirm the problem looking at it right now.

**Comment by gAlleb** [comment]
I've tested it up to 2.2.1 - problem persists.

**Comment by toots** [comment]
There should be a fix in the latest `rolling-release-v2.2.x` and `main`!
